templates/home/index.html

```diff
@@ -384,25 +384,39 @@ <h1 data-aos="fade-up">Donate Us</h1>
 <!-- Donate Us Modal begins -->
 <div id="don-modal">
     <div class="don-modal-contents">
-        <span class="close" id="don-close">&times</span>
+        <span class="close" id="don-close"><i class="fas fa-times"> </i></span>
         <div class="don-box1">
-            <h2>Donate Us</h2>
+                <h1>DONATE</h1>        
         </div>
         <div class="container">
             <div class="don-box2">
-                <h3>Bank Details</h3>
-                <h4>Account Name</h4>
-                <p>JAGRATI</p>
-                <h4>Account No.</h4>
-                <p>xxxxxxxxxxx</p>
-                <h4>IFSC Code</h4>
-                <p>XXXXxxxxxxx</p>
+                <div class="grid-container">
+                    <h4>Account Name :</h4>
+                    <p>JAGRATI</p>
+                    <h4>Account No :</h4>
+                    <p>xxxxxxxxxxx</p>
+                    <h4>IFSC Code :</h4>
+                    <p>XXXXxxxxxxx</p>
+                </div>
+
             </div>
             <div class="don-box3">
-                <h3>Contact Details</h3>
-                <p><i class="fas fa-mobile fa-2x"></i> +91-xxxxx-xxxxx</p>
-                <p><i class="fas fa-envelope fa-2x"></i> jagrati@iiitdmj.ac.in</p>
-                <p><i class="fas fa-map-marker-alt fa-2x"></i> PDPM IIITDM JABALPUR</p>
+                <p class="don-box3-p"><span>OR</span></p>
```

Comment:
Use `or` class name here, which is more specific and easy to see on which element this class name might be used.